public/app/types/explore.ts

```diff
@@ -241,7 +241,7 @@ export type RichHistoryQuery = {
   datasourceId: string;
   starred: boolean;
   comment: string;
-  queries: string[];
+  queries: DataQuery[];
```

Comment:
Also here it is probably more correct to say this is `(DataQuery | string)[]` I assume that may ripple out a bit and we will need to check what it really is but we already need to do that.

Reply:
✅ it is correct, as we now only receive DataQuery, no strings.